tests/Integration/Transport/HttpTest.php

```diff
@@ -97,6 +97,6 @@ public function testSilentlySendTraces()
         $httpTransport->send($traces);
         $output = ob_get_clean();
 
-        $this->assertEmpty($output);
+        $this->assertSame('', $output);
```

Comment:
Probably better to compare with an empty string, as 

```php
$this->assertEmpty('[]');
```

will pass.

Reply:
Could check strlen() also, but this is fine imo.